arangod/RestHandler/RestVersionHandler.cpp

```diff
@@ -88,6 +88,8 @@ RestStatus RestVersionHandler::execute() {
     }  // found
   }    // allowInfo
   result.close();
+  response()->setAllowCompression(true);
```

Comment:
I am wondering if this makes sense here. The version response typically is not very large.

Reply:
This is/was for testing only.

Reply:
Ok, let's remove it anyway there.